index.js

```diff
@@ -1,39 +1,171 @@
-const readline = require('readline');
-const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+const readline = require("readline");
+const readlineInterface = readline.createInterface(
+	process.stdin,
+	process.stdout
+);
 
 function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    readlineInterface.question(questionText, resolve);
-  });
+	return new Promise((resolve, reject) => {
+		readlineInterface.question(questionText, resolve);
+	});
 }
 
 // remember the StateMachine lecture
 // https://bootcamp.burlingtoncodeacademy.com/lessons/cs/state-machines
-let states = {
-  'roomOne': { canChangeTo: [ 'roomTwo' ] },
-  'roomTwo': { canChangeTo: [ 'roomThree' ] },
-  'roomThree': { canChangeTo: [ 'roomOne' ] }
+let states = { // I'm sure I need functions inside this for change the inventory of the room?
+	"182main": {
+		canChangeTo: ["foyer", "mr.mikes", "muddys"],
+		name: "182 Main St.",
+		welcomeMessage: `
+182 Main St. 
+You are standing on Main Street between Church and South Winooski.
+There is a door here. A keypad sits on the handle.
+On the door is a handwritten sign.
+
+`,
+		readSign: `
+
+The sign says "Welcome to Burlington Code Academy! Come on 
+up to the third floor. If the door is locked, use the code
+12345."
+
+`
+	},
+	"foyer": {
+		canChangeTo: ["182main", "stairs"],
+		name: "Foyer",
+		welcomeMessage: `
+You are in a foyer. Or maybe it's an antechamber. Or a 
+vestibule. Or an entryway. Or an atrium. Or a narthex.
+But let's forget all that fancy flatlander vocabulary,
+and just call it a foyer. In Vermont, this is pronounced
+"FO-ee-yurr".
+
+A copy of Seven Days lies in a corner.
+
+`,
+		readPaper: `
+You pick up the paper and leaf through it looking for comics 
+and ignoring the articles, just like everybody else does.
+
+`,
+		takePaper: `
+You are carrying:
```

Comment:
This will output the proper text to the screen, but won't actually add the paper to the player's inventory. Consider using a more generic 'pickup' method that will take any item passed in as an argument, and if it is takeable, add it to the player's inventory.